vsintegration/update-vsintegration.cmd

```diff
@@ -138,15 +138,6 @@ if /i "%PROCESSOR_ARCHITECTURE%"=="AMD64" (
     %SN64% -Vr VisualFSharp.Salsa,b03f5f7f11d50a3a
 )
 
-if exist "%ProgramFiles(x86)%\Microsoft Visual Studio %VisualStudioVersion%\Common7\IDE\PrivateAssemblies" (
```

Comment:
With this PR, the build output will contain two things:

1. FSharpSDK.msi: which installs the fsc/fsi bits in into SDK folder, and add the reference assemblies.
2. VisualsFSharp.vsix: which contains VS stuff.

The first can work independently without the second, and the second can be installed/uninstalled from VS without damaging VS setup. But we still need that script file to update the compiler bits, which are installed with the msi.